pkg/domain/infosync/region.go

```diff
@@ -78,7 +78,31 @@ func GetRegionDistributionByKeyRange(ctx context.Context, startKey []byte, endKe
 		return nil, err
 	}
 	if is.pdHTTPCli == nil {
-		return nil, errs.ErrClientGetLeader.FastGenByArgs("pd client not found")
+		return nil, errs.ErrClientGetLeader.FastGenByArgs("http cli is nil")
```

Comment:
Is this error appropriate?

Reply:
revert

Reply:
Why use this error(ErrClientGetLeader)? 😂

Reply:
The HTTP client will init after tidb connects the pd leader, so if the client is nil, it means that the tidb can't get the PD leader